Source/SvgElementIdManager.cs

```diff
@@ -153,15 +153,6 @@ public string EnsureValidId(string id, bool autoForceUniqueID = false)
                 return id;
             }
 
-            if (char.IsDigit(id[0]))

```

Comment:
Please adapt the documentation:
- Ensures that the specified ID is unique (instead of valid, as validity is not checked anymore)
- remove: The ID cannot start with a digit.